### PR TITLE
Add keyboard "purin" to qmk firmware

### PR DESCRIPTION
**Pull request by oRE-o**
## Description

Hello! I'm oREo, and it is my first time GitHub Pull Request!
I developed a PCB for a first time, and I want to contribute to qmk.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/purin/config.h
Comment (dunk2k) [inline keyboards/purin/config.h:1]: This file provides no functionality, can it please be removed.

File: keyboards/purin/readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# purin
+
+![purin](https://i.imgur.com/ig8DWEF.jpg)
```
Comment (dunk2k) [inline keyboards/purin/readme.md:3]:
```suggestion
![purin](https://i.imgur.com/ig8DWEFh.jpg)
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/purin/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+     * ┌───┬───┬───┬───┐
+     * │ 7 │ 8 │ 9 │ / │
+     * ├───┼───┼───┼───┤
+     * │ 4 │ 5 │ 6 │ * │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │ - │
+     * ├───┼───┼───┼───┤
+     * │ 0 │ . │Ent│ + │
+     * └───┴───┴───┴───┘
+     */
```
Comment (dunk2k) [inline keyboards/purin/keymaps/via/keymap.c:17]:
```suggestion
```

File: keyboards/purin/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+     * ┌───┬───┬───┬───┐
+     * │ 7 │ 8 │ 9 │ / │
+     * ├───┼───┼───┼───┤
+     * │ 4 │ 5 │ 6 │ * │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │ - │
+     * ├───┼───┼───┼───┤
+     * │ 0 │ . │Ent│ + │
+     * └───┴───┴───┴───┘
+     */
```
Comment (dunk2k) [inline keyboards/purin/keymaps/default/keymap.c:17]:
```suggestion
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/purin/config.h
Comment (oRE-o) [reply keyboards/purin/config.h:1]: Thank you for helping me!

File: keyboards/purin/readme.md (hunk 1 above)
Comment (oRE-o) [reply keyboards/purin/readme.md]: Thank you! but I have a small question (using the imgur for the first time), what is the difference between using h or not?

Comment (dunk2k) [reply keyboards/purin/readme.md]: The h resizes the image

For reference:
```
s =   90×  90 = Small Square (as seen in the example above)
b =  160× 160 = Big Square 
t =  160× 160 = Small Thumbnail 
m =  320× 320 = Medium Thumbnail 
l =  640× 640 = Large Thumbnail
h = 1024×1024 = Huge Thumbnail
```

**Comment by oRE-o** [comment]
Work complete! can It be merged?

**Comment by dunk2k** [comment]
> Work complete! can It be merged?

That would be for QMK collaborator(s) to review and merge. As there's no minimum time to review PRs, it's really a case of waiting until it's seen.

**Review by fauxpark — APPROVED** [review]

**Comment by oRE-o** [comment]
Thanks for helping me :) !

**Review by waffle87 — APPROVED** [review]

**Pull request merged**